oxford-metadata.ttl

```diff
@@ -356,33 +361,34 @@
 
 :membrane_stimulus_current
     a :Annotation ;
-    a :StimulusCurrent ;
+    a :StimulusCurrentRelated ;
```

Comment:
For the stimulus current do we also want `StimulusCurrent` (itself, this one) and `StimulusCurrentRelated`?